rotkehlchen/data/global.db.sqbpro

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?><sqlb_project><db path="/home/yabirgb/work/rotki/rotkehlchen/data/global.db" readonly="0" foreign_keys="1" case_sensitive_like="0" temp_store="0" wal_autocheckpoint="1000" synchronous="2"/><attached/><window><main_tabs open="structure browser pragmas query" current="3"/></window><tab_structure><column_width id="0" width="300"/><column_width id="1" width="0"/><column_width id="2" width="100"/><column_width id="3" width="3221"/><column_width id="4" width="0"/><expanded_item id="0" parent="1"/><expanded_item id="1" parent="1"/><expanded_item id="2" parent="1"/><expanded_item id="3" parent="1"/></tab_structure><tab_browse><current_table name="4,11:mainasset_types"/><default_encoding codec=""/><browse_table_settings/></tab_browse><tab_sql><sql name="SQL 1">INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES(&quot;0x2565ae0385659badCada1031DB704442E1b69982&quot;, 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES(&quot;_ceth_0x2565ae0385659badCada1031DB704442E1b69982&quot;, &quot;C&quot;, &quot;ASSEMBLE&quot;, &quot;ASM&quot;, 1589092948, NULL, &quot;assemble-protocol&quot;, &quot;ASM&quot;, &quot;0x2565ae0385659badCada1031DB704442E1b69982&quot;);
```

Comment:
ehm .... wtf is this? Don't commit this file.

Reply:
I don't know really